src/main/scala/com/gu/support/catalog/ProductRatePlan.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.support.catalog
+
+import com.gu.i18n.Currency.GBP
+
+case class ProductRatePlan(
+  id: String,
+  status: Status,
+  name: Option[String],
+  productRatePlanCharges: List[ProductRatePlanCharge]
+) {
+  def price: Double = {
+    productRatePlanCharges
+      .filter(_.endDateCondition.contains("Subscription_End"))
```

Comment:
This will exclude the 6 for 6 charge - is that OK?

Reply:
See comment above, I'll talk to you offline about 6 for 6 because I need to know a bit more about how it's modelled